doc/source/configuration/vault.rst

```diff
@@ -36,6 +36,8 @@ Before beginning the deployment of vault for openstack internal TLS and backend
 
   * Seed Node or a host to run the vault container on
   * Overcloud controller hosts to install second vault on
+  * Ansible Galaxy dependencies installed: ``ansible-galaxy install ansible-galaxy install -r $KAYOBE_CONFIG_PATH/anisble/requirements.yml``
+  * Python dependencies installed: ``pip install -r kayobe-config/requirements.txt``
```

Comment:
This one's a bit awkward, I wish we had a cleaner way to pull it in.

Reply:
post hook on control host bootstrap?

Reply:
Could do. It also includes Kayobe though, and I'd be wary of overwriting an existing installation by surprise. Would ideally like something more "official". Will tried to do it years ago but gave up for reasons I forget. I think this is ok for now.